app/assets/stylesheets/header.scss

```diff
@@ -0,0 +1,9 @@
+.navbar .container {
+  .glyphicon-plus {
+    color: #9d9d9d;
```

Comment:
Color literals like `#9d9d9d` should only be used in variable declarations; they should be referred to via variable everywhere else.